templates/debian-default-autofs.erb

```diff
@@ -0,0 +1,145 @@
+#
```

Comment:
This template looks almost the same as for EL. It would be better to use one template instead of two.